src/Components/aliaa/Card.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./Card.css";
+
+function Card() {
+  return <section className="card-container">
```

Comment:
As a shortcut and to reduce the implementation number of lines we can replace this

```
 const Card = () => {
  return()
}
```
with this
`const Card = () => ()`